client-ui/src/app/components/lineage/lineage-details/operation-properties-details/operation-properties-details.component.ts

```diff
@@ -23,8 +23,9 @@ import { Property, PropertyType } from 'src/app/model/property';
 import { OperationType, PropertiesComponents } from 'src/app/model/types/operationType';
 import { AttributeVM } from 'src/app/model/viewModels/attributeVM';
 import { OperationDetailsVM } from 'src/app/model/viewModels/operationDetailsVM';
-import { getOperationIcon, getOperationColor } from 'src/app/util/execution-plan';
+import { getOperationColor, getOperationIcon } from 'src/app/util/execution-plan';
 import { getText } from 'src/app/util/expressions';
+import { PropertiesComponent } from './properties/properties.component';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [' should be "](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4600414)